packages/addons/README.md

```diff
@@ -62,16 +62,29 @@ Convenient tools to know the type/kind of BPMN elements.
 
 ### `BpmnElementsSearcher`
 
-`bpmn-visualization`only provides APIs that take BPMN element ids as parameters.
+`bpmn-visualization` only provides APIs that take BPMN element ids as parameters.
 However, there are scenario where the ids in the BPMN source/model have been generated and are unknown by the application.
 
-Instead, the app knows the name of the elements. `BpmnElementsSearcher` provides a way to retrieve the ids of elements related to their names.
+Instead, the application knows the name of the elements. `BpmnElementsSearcher` provides a way to retrieve elements given their names.
 
 This is useful for example in "Process Discovery" scenario. The elements are identified by name only. Ids may be generated
 and not fully linked with the name of the elements.
 
+IMPORTANT: There is no guarantee that names are unique in the BPMN source. In case that there are several matches, `BpmnElementsSearcher` is doing filtering (see below).
```

Comment:
```suggestion
IMPORTANT: There is no guarantee that names are unique in the BPMN source. In case that there are several matches, `BpmnElementsSearcher` may do filtering (see below).
```